build.gradle

```diff
@@ -23,7 +23,7 @@ dependencies {
     compile  "org.embulk:embulk-core:0.9.8"
     provided "org.embulk:embulk-core:0.9.8"
 
-    compile "com.microsoft.azure:azure-storage:8.0.0"
+    compile "com.microsoft.azure:azure-storage:8.6.6"
```

Comment:
be aware this version use  jackson-core 2.9.4

Reply:
Tested, no conflict so far with embulk v0.9.18 and v0.10.28